docs/general/emulator-support-and-issues.md

```diff
@@ -456,6 +461,7 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 - ❌ Standalone emulator: **AetherSX2**
   - No longer being developed, has known incompatibilities
 - ❌ Standalone emulator: **NetherSX2**
+  - Mod of the above emulator
```

Comment:
This should read "Fork of AetherSX2, known to have many issues with achievements and hardcore mode".